uwrt_mars_rover_utils/uwrt_mars_rover_xbox_controller/include/uwrt_mars_rover_xbox_controller/coordinateNode.hpp

```diff
@@ -0,0 +1,25 @@
+#ifndef COORDINATENODE_HPP_
+#define COORDINATENODE_HPP_
+
+#include <uwrt_mars_rover_xbox_controller/visibility.h>
+
+#include <geometry_msgs/msg/twist.hpp>
+#include <rclcpp/rclcpp.hpp>
+#include <uwrt_mars_rover_xbox_controller/msg/xbox_controller.hpp>
+
+namespace drivetraincontrollerComposition
+{
+class CoordinateNode : public rclcpp::Node
+{
+public:
+  UWRT_MARS_ROVER_XBOX_CONTROLLER_PUBLIC
+  explicit CoordinateNode(const rclcpp::NodeOptions & options);
+
+private:
+  rclcpp::Publisher<geometry_msgs::msg::Twist>::SharedPtr pub_;
```

Comment:
make this more explicit; proper variable naming is important. Also be sure to change this anywhere else that is appropriate. 
```suggestion
  rclcpp::Publisher<geometry_msgs::msg::Twist>::SharedPtr joystick_vel_pub_;
```